plugins/plugin_utils/_hashi_vault_lookup_base.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 # Copyright (c) 2021 Brian Scholer (@briantist)
-# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
The GPL license file is called `LICENSE`, not `COPYING`, in this collection. (Might be wrong in more files?)

Reply:
good catch thanks, I think the other files are correct, and that I just copied this from the wrong place, but I will check on that. Anyway if I implement the move to a directory, I'll also have to revisit all the headers.

Reply:
Actually the other files are all wrong lol

Reply:
This will be addressed in part 2:
- #282

Reply:
I think this is a quite common mistake, many collections have that problem since the "big split" for Ansible 2.10. The collection template and the collection repos created back then were using `LICENSE` for the filename, while the comments in all modules/plugins migrated from ansible/ansible were referencing it as `COPYING`. I renamed `LICENSE` to `COPYING` in the collections I was working on (like community.general) to avoid changing all the headers, other collections adjust the headers, or didn't change anything ;-)